x-pack/plugins/ml/public/application/explorer/explorer.js

```diff
@@ -257,10 +261,14 @@ export class Explorer extends React.Component {
       );
     }
 
-    if (noJobsFound && hasResults === false && !loading) {
+    if (hasResultsWithAnomalies === false && !loading) {
       return (
         <ExplorerPage jobSelectorProps={jobSelectorProps}>
-          <ExplorerNoResultsFound />
+          <ExplorerNoResultsFound
+            hasResults={hasResults}
+            hasResultsWithAnomalies={hasResultsWithAnomalies}
+            selectedJobsRunning={selectedJobsRunning}
+          />
```

Comment:
Since `hasResultsWithAnomalies` has the hard coded check against `false` here, it might not be necessary to pass it on as a prop to `ExplorerNoResultsFound` since we can assume within the component it will always be false..

Reply:
Good call - missed removing it. 👍  Updated in 1e10d7a3019dc185f328e5ae74cb74fe9c0f83e0